api/envoy/extensions/http/custom_response/redirect_policy/v3/redirect_policy.proto

```diff
@@ -26,28 +26,75 @@ option (xds.annotations.v3.file_status).work_in_progress = true;
 // upstream.
 // [#next-free-field: 7]
 message RedirectPolicy {
-  // [#comment: TODO(pradeepcrao): Add the ability to specify the full uri, or just host or
-  // path rewrite for the redirection in the same vein as
-  // config.route.v3.RedirectAction]
-  // The host that will serve the custom response.
-  //
-  // Example:
-  //
-  // .. code-block:: yaml
-  //
-  //    uri: https://www.mydomain.com
-  //
-  string host = 1 [(validate.rules).string = {min_len: 1}];
-
-  // The path for the custom response.
-  //
-  // Example:
-  //
-  //  .. code-block:: yaml
-  //
-  //    path: /path/to/503_response.txt
-  //
-  string path = 2 [(validate.rules).string = {min_len: 1}];
+  // [#next-free-field: 7]
```

Comment:
Can we reuse config.route.v3.RedirectAction here, or is it different enough that a new config is needed?

Reply:
The main difference is that certain fields such as `response_code`, `prefix_rewrite` and `regex_rewrite` don't make sense for custom response. `response_code` does not make sense because it's limited to redirect responses, and custom response already has a separate field for it. The other two don't make sense because we don't have access to the original request before `RedirectAction` was applied to it by the route, as well as the original route itself, to infer how to apply these fields for custom response.